data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/accumulator/BufferTypeOptions.java

```diff
@@ -5,13 +5,34 @@
 
 package org.opensearch.dataprepper.plugins.sink.accumulator;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 /**
  * Defines all the buffer types enumerations.
  */
 public enum BufferTypeOptions {
 
-    INMEMORY,
-    LOCALFILE
+    LOCALFILE("local_file", new LocalFileBuffer());
```

Comment:
I thought the plan is to support both INMEMORY and LOCALFILE. Isn't it?

Reply:
We are supporting both INMEMORY and LOCALFILE buffer type. The INMEMORY buffer functionality was raised as separate PR: https://github.com/opensearch-project/data-prepper/pull/2623